### PR TITLE
OSOE-548: Upgrade to Orchard Core 1.6 in Orchard-Vue.js

### PR DESCRIPTION
**Pull request by Psichorex**
[OSOE-548](https://lombiq.atlassian.net/browse/OSOE-548)

[OSOE-548]: https://lombiq.atlassian.net/browse/OSOE-548?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Lombiq.VueJs/Lombiq.VueJs.csproj [hunk 1]
```diff
@@ -25,30 +25,20 @@
...
-    <None Include="Assets/**" Pack="true" PackagePath="Assets/" />
-    <None Remove="node_modules/**" />
 
-    <!-- Necessary to include the reusabe SFCs in this module. For other modules that consume Lombiq.VueJs this is
-         handled by the Lombiq.VueJs.props file. -->
-    <EmbeddedResource Include="Assets\Scripts\VueComponents\*.vue" />
-
-    <!-- Necessary to avoid NU5118 due to duplicate build/Lombiq.VueJs.props files. The referenced module
-         "OrchardCore.Module.Targets" tries to generate that file from the Package.Build.props mentioned below. -->
-    <None Remove="$(NuGetPackageRoot)orchardcore.module.targets\*\build\Package.Build.props" />
```
Comment (Piedone) [inline Lombiq.VueJs/Lombiq.VueJs.csproj]: Why are all these changes necessary?

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.VueJs/Lombiq.VueJs.csproj (hunk 1 above)
Comment (Psichorex) [reply Lombiq.VueJs/Lombiq.VueJs.csproj]: Wasn't intentional. Reverted.

**Review by Piedone — APPROVED** [review]

File: Lombiq.VueJs.Samples/Lombiq.VueJs.Samples.csproj [hunk 2]
```diff
@@ -42,12 +42,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.Common\Lombiq.HelpfulLibraries.Common.csproj" />
+    <ProjectReference Condition="'$(NuGetBuild)' != 'true'" Include="..\..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.Common\Lombiq.HelpfulLibraries.Common.csproj" />
```
Comment (Piedone) [inline Lombiq.VueJs.Samples/Lombiq.VueJs.Samples.csproj]: Rather add a conditional `ItemGroup` like in other projects for distinction.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.VueJs.Samples/Lombiq.VueJs.Samples.csproj (hunk 2 above)
Comment (Psichorex) [reply Lombiq.VueJs.Samples/Lombiq.VueJs.Samples.csproj]: Using conditional `ItemGroup` kill the project for me actually:
image

Comment (Piedone) [reply Lombiq.VueJs.Samples/Lombiq.VueJs.Samples.csproj]: It works in dozens of projects, including here a few lines below, so please make sure you're doing it correctly.

Comment (Psichorex) [reply Lombiq.VueJs.Samples/Lombiq.VueJs.Samples.csproj]: According to the MSB4232 you can't include something in an `ItemGroup` that doesn not have `Include, Update,Remove`.
We are only trying to `Import Project=` thus why it gives an error and the project won't load.
Neither of the projects have Import elements in ItemGroups.
`<Import>` is actually a oneliner in every single usecase in our projects.

Comment (Piedone) [reply Lombiq.VueJs.Samples/Lombiq.VueJs.Samples.csproj]: Lombiq.Hosting.Tenants.FeaturesGuard has exactly what you need:

```
  <ItemGroup Condition="'$(NuGetBuild)' != 'true'">
    <ProjectReference Include="..\..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.Common\Lombiq.HelpfulLibraries.Common.csproj" />
    <ProjectReference Include="..\..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.OrchardCore\Lombiq.HelpfulLibraries.OrchardCore.csproj" />
  </ItemGroup>
```

**Review by Piedone — APPROVED** [review]

**Review by Piedone — APPROVED** [review]
Just marking the merge from `issue/OSOE-641`.

**Pull request merged**